oci/oci.go

```diff
@@ -14,53 +15,80 @@ import (
 	spec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
-// Init initializes the img directory as an OCI image.
-// i.e. Creates oci-layout, index.json, and blobs.
-//
-// img directory must not exist before calling this function.
+// BlobWriter writes an OCI blob and returns a digest when closed.
+type BlobWriter interface {
```

Comment:
@AkihiroSuda The content package is the premiere approach for doing io on OCI oriented content. Use the interface definition, don't define a new one that is slightly different.